azcollectc.js

```diff
@@ -80,18 +80,11 @@ class AzcollectC extends AlServiceC {
 
     _doCheckinAws(checkinValues) {
```

Comment:
Should we use some sort of default value for this incase a collector doesnt pass it in. I'm thinking some sort of delinquent guard duty collector.

Reply:
nope. in case of error a collector will get bar request error from Azcollect and it should be fixed on either side: azcollect or collector itself.